pkg/server/restart.go

```diff
@@ -101,6 +103,11 @@ func (c *criContainerdService) recover(ctx context.Context) error {
 		return fmt.Errorf("failed to load images: %v", err)
 	}
 	for _, image := range images {
+		if _, err := c.client.SnapshotService(c.config.ContainerdConfig.Snapshotter).Stat(ctx, image.ChainID); err != nil {
+			glog.Warningf("Failed to check top snapshot %+v: %v", image, err)
```

Comment:
top snapshot?

Reply:
Thanks. What do you suggest?

Reply:
"Failed to stat snapshot for image %+v: %v"
or 
"Failed to locate information for image %+v: %v"

Reply:
We want to check readiness of top snapshot.  We do it by calling Stat() function now.
Is the error message better to indicating our purpose?

Reply:
I understand your meaning and the issue description but most won't.  Top snapshot could have many meanings. Images have layers.. the image.chainid points to the top-level layer for an image snapshot and contains references to the rest of the layers. IMO it's better to just say what failed vs. using the issue description of what needed to be done here.  

For example:  "Failed to locate information for snapshot using chainid for image %+v: %v"
As another example: "Failed to stat the top-level snapshot for image %+v: %v"
I think those are more explanatory to the reader of the warnings than: "Failed to check top snapshot %+v: %v"

What do you think?

Reply:
You are right. You have a lot of experience,we need to learn from you.:)
Thank for your review and guidance.

Reply:
done